### PR TITLE
Add topic in reuse search filter and meta information

### PR DESCRIPTION
**Pull request by maudetes**
Fix https://github.com/etalab/data.gouv.fr/issues/638

Follows https://github.com/opendatateam/udata/pull/2689. It allows to display the topic information on reuse page in the meta-information tab if specified.
It also changes actual `type` filter in reuse search page by topic instead.

### CODE REVIEW

**Comment by maudetes** [comment]
We can either keep the actual `tabs` or change them for DSFR or actual custom `tags`.

Alternative for tags using DSFR with custom active tag
image

Alternative for tags using actual tags with a new custom active tag
image

**Comment by agarrone** [comment]
Exemple : https://www.gouvernement.fr/toute-l-actualite

File: theme/less/components/tabs.less [hunk 1]
```diff
@@ -63,11 +63,10 @@ You can use tabs from DSFR. It uses DSFR colors and conventions. It allows you t
 
 .tabs {
     display: flex;
+    flex-wrap: wrap;
 
     .tab {
-        + .tab {
-            margin-left: @spacing-sm;
-        }
+        margin: @spacing-xxs;
```
Comment (nicolaskempf57) [inline theme/less/components/tabs.less:69]: We can remove this line here and use DSFR utility class on reuse/list and home to allow for different margins (`.fr-m-1v` on reuse/list and `.fr-ml-3v` on home to keep the previous layout).

File: theme/less/components/tabs.less [hunk 2]
```diff
@@ -63,11 +63,10 @@ You can use tabs from DSFR. It uses DSFR colors and conventions. It allows you t
...
 
         //Prevents children from taking the click events.
```
Comment (nicolaskempf57) [inline theme/less/components/tabs.less:71]:
```suggestion
        // Prevents children from taking the click events.
```

File: theme/less/components/tabs.less [hunk 3]
```diff
@@ -63,11 +63,10 @@ You can use tabs from DSFR. It uses DSFR colors and conventions. It allows you t
...
         //Might be dangerous, remove if needed
```
Comment (nicolaskempf57) [inline theme/less/components/tabs.less:72]:
```suggestion
        // Might be dangerous, remove if needed
```

**Review by nicolaskempf57 — COMMENTED** [review]
We can use DSFR utilities for margin/padding instead of our own spacing system so hopefully we'll be able to remove our custom utility classes in the future.

File: theme/less/components/tabs.less (hunk 2 above)
Comment (maudetes) [reply theme/less/components/tabs.less:71]: Will be fixed in https://github.com/etalab/udata-front/pull/57

File: theme/less/components/tabs.less (hunk 3 above)
Comment (maudetes) [reply theme/less/components/tabs.less:72]: Will be fixed in https://github.com/etalab/udata-front/pull/57

File: theme/less/components/tabs.less (hunk 1 above)
Comment (maudetes) [reply theme/less/components/tabs.less:69]: We'll use DSFR tags instead following https://github.com/etalab/udata-front/pull/57.

**Review by nicolaskempf57 — APPROVED** [review]

**Pull request merged**